### PR TITLE
Mpuncel/ds audit records 2

### PR DESCRIPTION
**Pull request by mpuncel**

### CODE REVIEW

File: pkg/store/consul/dsstore/auditing_store.go [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+	"github.com/square/p2/pkg/audit"
+	"github.com/square/p2/pkg/ds/fields"
+	"github.com/square/p2/pkg/manifest"
+	"github.com/square/p2/pkg/types"
+	"github.com/square/p2/pkg/util"
+
+	klabels "k8s.io/kubernetes/pkg/labels"
+)
+
+type AuditLogStore interface {
```
Comment (rudle) [inline pkg/store/consul/dsstore/auditing_store.go]: Does this need to be public?

Comment (mpuncel) [inline pkg/store/consul/dsstore/auditing_store.go]: hmm it probably doesn't have to be, although i guess if we pretended we had godocs i don't think private types get documented?

**Comment by mpuncel** [comment]
the stuff after the approval is just a rebase so i'm still considering this approved

**Comment by mpuncel** [comment]
had to fix up some Commit() calls in tests

**Pull request merged**